web/src/pages/core/public/PrivacyPolicy/modal.css

```diff
@@ -0,0 +1,32 @@
+.modal {
```

Comment:
We dont use pure css on anubis.

Look at some other pages and you will see we use the material ui makeStyles hook to make a dynamic style sheet. 
https://v4.mui.com/styles/api/#makestyles-styles-options-hook